### PR TITLE
website/docs: move Sources from Integrations into Docs

### PR DESCRIPTION
**Pull request by tanberry**
This PR moves all of the Sources docs over to the Docs menu, out of the Integrations, and cleans up the LDAP Source page.
-- moved `/sources` directory
-- updated `sidebars.js` and `sidebarsIntegrations/js`
-- updated the `netlify.toml` file to handle redirects for any incoming, external links
-- fixed broken links between files
-- reorganized all the Settings on the LDAP Source page to reflect the UI... [TODO] still need to clarify the **Synch parent group** field (I don't see it on the UI).

UPDATE: I added a new landing page for Integrations, that links to both Applications and to Sources.

-   [x] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/developer-docs/docs/templates/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The most common types are:
 
 ### Add a new integration
 
-To add documentation for a new [integration](../../../integrations/) (with support level Community or Vendor), please use the integration template [`service.md`](https://github.com/goauthentik/authentik/blob/main/website/integrations/_template/service.md) file from our GitHub repo. You can download the template file using the following command:
+To add documentation for a new integration (with support level Community or Vendor), please use the integration template [`service.md`](https://github.com/goauthentik/authentik/blob/main/website/integrations/_template/service.md) file from our GitHub repo. You can download the template file using the following command:
```
Comment (kensternberg-authentik) [inline website/developer-docs/docs/templates/index.md:19]: Nitpick: the word "file" in "... file from out GitHub repo" is redundant.  A template is a file.  

Also, I'm not entirely sure I want a user who needs help using wget using our product.  Most people will just click the link and do "Save as...", I suspect.

File: website/docs/sources/general.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Overview of sources
+slug: general
+---
+
+Sources allow you to connect authentik to an external user directory. They can also be used for social logins, using external providers such as Facebook, Twitter, etc.
```
Comment (kensternberg-authentik) [inline website/docs/sources/general.md:6]: The commas here feel... off-balanced?  Perhaps "Sources can also be used with social login providers such as Facebook, Twitter, or GitHub."

File: website/docs/sources/ldap/index.md [hunk 3]
```diff
@@ -2,41 +2,74 @@
...
--   **Sync groups**: Enable/disable group synchronization. Groups are synced in the background every 5 minutes.
--   **Sync parent group**: Optionally set this group as the parent group for all synced groups. An example use case of this would be to import Active Directory groups under a root `imported-from-ad` group.
--   **Property mappings** and **Group Property Mappings**: Define which LDAP properties map to which authentik properties. The default set of property mappings is generated for Active Directory. See also [LDAP Property Mappings](../../../docs/property-mappings/#ldap-property-mapping)
 
 ## Property mappings
 
 LDAP property mappings can be used to convert the raw LDAP response into an authentik user/group.
 
-By default, authentik ships with some pre-configured mappings for the most common LDAP setups.
+By default, authentik ships with some [pre-configured mappings](../../property-mappings/index.md#ldap-property-mapping) for the most common LDAP setups.
```
Comment (kensternberg-authentik) [inline website/docs/sources/ldap/index.md:72]: Impression: "some" weakens the impact.  Removing it makes the sentence clear:

"authentik ships with pre-configured mappings for the most common LDAP setups."

(... "These mappings can be found on the LDAP Source Configuration page.", maybe?)

**Review by kensternberg-authentik — APPROVED** [review]
More nitpicking.  Otherwise fine.

**Review by rissson — APPROVED** [review]
I'll approve in principal, not sure I'll have time to review the whole text soon

File: website/developer-docs/docs/templates/index.md (hunk 1 above)
Comment (tanberry) [reply website/developer-docs/docs/templates/index.md:19]: I removed "file" but left the `wget` command... I want our users to learn and also make sure we are friendly to new users.

**Pull request merged**